README.md

```diff
@@ -227,7 +227,7 @@ changed, then deliver the new handler to the promise which unblocks
 the server.
 
 Note that we only need to go to this additional effort if retaining
-open resources is useful during development, otherwise we can rely on
+open resources is useful during development, otherwise, we can rely on
```

Comment:
I think the comma you've placed here isn't correct.

Reply:
Removed it